frontend/src/pages/DevicesPage/DevicesPage.tsx

```diff
@@ -47,6 +56,7 @@ export const DevicesPage: React.FC<Props> = ({ restore, select }) => {
           select={select}
         />
       )}
+      {isPortal() && (<DialogNewFeatures open={open} onClose={onClose} />)}
```

Comment:
Any visibility or open/close state should be in the component to keep this one as simple as possible.
```suggestion
      <DialogNewFeatures />
```